scripts/deploy-on-kind.sh

```diff
@@ -231,6 +231,7 @@ function install_dependencies() {
       export DOCKER_SERVER="localregistry-docker-registry.default.svc.cluster.local:30050"
       export DOCKER_USERNAME="whatevs"
       export DOCKER_PASSWORD="whatevs"
+      export KPACK_TAG="localregistry-docker-registry.default.svc.cluster.local:30050/cf-relint-greengrass/cf-k8s-controllers/kpack/beta"
```

Comment:
We could probably use a shorter/cleaner tag here, if we're making the change to replace the whole repo tag. We definitely don't need all that cf-relint-greengrass stuff in there when we are storing things locally. We just left them in before because the local registry doesn't care, and it was the easiest `sed` command to write.

Reply:
Interestingly, kpack breaks if this doesn't end in 'kpack/beta' and does have at least three slash separated items.

Btw, this is just a test pr to trigger the new pipeline.